sound-bubble/imports/ui/components/Account/UserFeed.jsx

```diff
@@ -35,7 +44,9 @@ class UserFeed extends Component {
           <ul>{this.getSongDetails()}</ul>
         </div>
         <div className="show_more_button_container">
-          <button className="btn btn-secondary btn-sm"> Show More </button>
+          <button className="btn btn-secondary btn-sm" onClick={this.loadMore}>
+            Show More
+          </button>
```

Comment:
So are we doing show more button or just scroll down?

Reply:
Which one do you prefer? I haven't implemented scroll down yet cuz we haven't reached a consensus.